### PR TITLE
adding regression test for suite RRFS_v1beta

### PR DESCRIPTION
**Pull request by panll**
added file:
     tests/tests/fv3_ccpp_rrfs_v1beta
modified file:
     tests/rt_ccpp_gsd.conf

### CODE REVIEW

File: tests/rt_ccpp_gsd.conf [hunk 1]
```diff
@@ -3,17 +3,23 @@
...
 RUN     | fv3_ccpp_gf_thompson                                                                                                      | standard    |                | fv3         |
 RUN     | fv3_ccpp_gsd                                                                                                              | standard    |                | fv3         |
 RUN     | fv3_ccpp_gsd_coldstart                                                                                                    | standard    |                |             |
 RUN     | fv3_ccpp_gsd_warmstart                                                                                                    | standard    |                |             | fv3_ccpp_gsd_coldstart
 RUN     | fv3_ccpp_gsd_noah                                                                                                         | standard    |                | fv3         |
+# Compile with CCPP - static mode
+COMPILE | CCPP=Y REPRO=Y SUITES=FV3_GSD_SAR_v1,FV3_RRFS_v0, FV3_RRFS_v1beta                                                         | standard    | hera.intel     | fv3         |
```
Comment (climbfuji) [inline tests/rt_ccpp_gsd.conf:17]: This will not work - no spaces in the SUITES=... argument. Have you tried it?

**Comment by panll** [comment]
Thanks, Dom! The space is removed and the file is updated. It's tested on hera. @climbfuji

**Comment by climbfuji** [comment]
This does not work without making the corresponding changes / additions in fv3atm. Please add the new SDF for RRFSv1beta, correct the old mynnrad pre/post calls and call order to the new sgscloud pre/post as discussed via email for all applicable suites, create a PR for fv3atm, and update .gitmodules and the fv3atm submodule pointer in ufs-weather-model to point to your fv3atm fork and branch.

See https://docs.google.com/presentation/d/1UVq7O0djhOO99VCIbftwi-WFRdF9fjnMijZNw4kRxnM/edit if this last bit is not clear.

Lastly, please run the tests on hera for `rt_ccpp_gsd.conf` (first create your own baseline to make sure the code compiles and all runs complete successfully, then verify against it to ensure run-to-run reproducibility). The above slides contain information on how to do this, instead of using `./rt.sh -f ...` use `./rt.sh -l rt_ccpp_gsd.conf -c -e` to create the baseline, and `./rt.sh -l rt_ccpp_gsd.conf -m -e` to verify against it.

**Comment by panll** [comment]
Thanks, Dom! I'll do the full regression test.

**Comment by panll** [comment]
here is the fv3atm pull request:
https://github.com/NCAR/fv3atm/pull/60
with corresponding suite file changes. @climbfuji

**Comment by panll** [comment]
I did the regression tests with rt_ccpp_gsd.conf and all passed on Hera. @climbfuji @JeffBeck-NOAA
 /scratch2/BMC/det/lpan/hera/v16/07092020/ufs-weather-model/tests/log_hera.intel>tail rt*.log |grep Test
Test 001 fv3_ccpp_thompson_mynn PASS
Test 002 fv3_ccpp_gf_thompson PASS
Test 003 fv3_ccpp_gsd PASS
Test 004 fv3_ccpp_gsd_coldstart PASS
Test 005 fv3_ccpp_gsd_warmstart PASS
Test 006 fv3_ccpp_gsd_noah PASS
Test 007 fv3_ccpp_rrfs_v0 PASS
Test 008 fv3_ccpp_rrfs_v1beta PASS
Test 009 fv3_ccpp_gsd_sar_v1 PASS
Test 010 fv3_ccpp_thompson PASS
Test 011 fv3_ccpp_gf PASS
Test 012 fv3_ccpp_mynn PASS
Test 013 fv3_ccpp_gsd_drag_suite PASS
Test 014 fv3_ccpp_gsd_sar PASS
Test 015 fv3_ccpp_gsd_debug PASS
Test 016 fv3_ccpp_gsd_sar_25km_debug PASS

**Comment by climbfuji** [comment]
Thanks for running the tests. I don't see the `.gitmodules` and `fv3atm` submodule pointer update in this `ufs-weather-model` PR. Since I am going to merge the fv3atm PR now, you need to update the submodule pointer in this PR before we can merge it.

**Comment by climbfuji** [comment]
PR https://github.com/NCAR/fv3atm/pull/60 is empty - no changes. Apparently you did not commit your changes.

**Comment by panll** [comment]
Sorry, could you check it again? I can see it from my side now.  Thanks! @climbfuji 
Screen Shot 2020-07-15 at 10 01 53 AM

**Comment by climbfuji** [comment]
Thanks, Linlin. I merged https://github.com/NCAR/fv3atm/pull/60 - forget about the .gitmodules change, no longer needed for this specific PR. Usually we require it before running the tests and conducting the code review, but that train has left.

Now you need to go to your FV3 directory, do a `git remote update` and check out the latest `dtc/develop`:
```
cd FV3
git remote update
git checkout origin/dtc/develop
git log
# you should see hash 96bb736c687bf886b0bd47e57df6658c96c8f5cf
cd ..
git status
# you should see a message the FV3 has new commits
git add FV3
git commit -m "Update submodule pointer for fv3atm"
git push YOUR_FORK YOUR_BRANCH_NAME
```

**Comment by panll** [comment]
Updated. Please let me know if anything else is needed, thanks! @climbfuji

**Pull request merged**